src/features/item-details/ItemProperties.js

```diff
@@ -35,6 +38,7 @@ const ItemProperties = ({
     url,
 }) => {
     const descriptionId = uniqueid('description_');
+    const shouldShowArchivedAt = isFeatureEnabled(features, 'details.archivedAt.enabled');
```

Comment:
Can we update the endpoint to not return the value if the feature isn't enabled yet, instead?